GoogleForCreators/web-stories-wp
# Safari text wrap hack

## Context
"Fixes" the Safari text wrap issue by reducing the font size in Safari by 0.5px.
Based on conversation with Will (who also reached out to UX), potential underflow is a preferred bug compared to the overflow that was the origin of this issue.

Thus, the decision was to reduce the font size in Safari so that it would eliminate the overflow option.



## Summary
- Adds a hack for reducing the font size in Safari in the outout.


## Relevant Technical Choices
Adds an additional `span` wrapper to the `p` since otherwise it's not guaranteed there's a `span` inside.


## To-do

- [x] Test more thoroughly



## User-facing changes



## Testing Instructions
See the Steps to Reproduce in the description of https://github.com/google/web-stories-wp/issues/6323.
Verify it's not happening anymore.


### QA
See the steps above.
## Checklist




---



Fixes #6323
